### DISCUSSION TITLE
Is it somehow possible to run 2 Actions on 1 Button?

### DISCUSSION THREAD

**Discussion by burntbread** [Windos/BurntToast#139] [Q&A]
Hello,

the Toast i would like to create needs 2 Actions after a single Button click
for example:

```
$Button1 = "Open"
$Link1 = "www.google.com"
$Link2 = "www.github.com"

$InArgs = @{
    Content   = $Button1
    Arguments = $Link , $Link2
}
$Button1 = New-BTButton @InArgs
```

but this wouldn't work because BurntToast only supports one Argument for each Button. How can i get around this issue? 
Is BurntToast maybe able to react to that Button click in a different way?

Thanks for your help

Kind regards

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
